scripts/data_plane/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+resources:
```

Comment:
This should an overlay in the `config` directory. 

Instead of the github URL, it should just a local file system reference to to `config/default`.